src/ray/stats/stats.h

```diff
@@ -18,53 +18,49 @@
 #include <string>
 #include <unordered_map>
 
-#include "opencensus/exporters/stats/prometheus/prometheus_exporter.h"
-#include "opencensus/exporters/stats/stdout/stdout_exporter.h"
 #include "opencensus/stats/stats.h"
 #include "opencensus/tags/tag_key.h"
-#include "prometheus/exposer.h"
 
+#include "ray/common/ray_config.h"
 #include "ray/stats/metric.h"
+#include "ray/stats/metric_exporter.h"
+#include "ray/stats/metric_exporter_client.h"
 #include "ray/util/logging.h"
 
 namespace ray {
 
 namespace stats {
 
+#include <boost/asio.hpp>
+
 /// Include metric_defs.h to define measure items.
 #include "metric_defs.h"
 
 /// Initialize stats.
-static void Init(const std::string &address, const TagsType &global_tags,
-                 bool disable_stats = false, bool enable_stdout_exporter = false) {
+static void Init(
```

Comment:
I just notice we have no shutdown action for stats module. Could you add this function where all opencensus exporters and timer handlers should be halted in.

Reply:
I didn't find any interface to shut the handler down. https://github.com/census-instrumentation/opencensus-cpp/blob/master/opencensus/stats/stats_exporter.h Do you happened to know what API I should use for this?

Reply:
Ah.. Probably no such API. But the clearHandlersForTesting method (in StatsExporterImpl) can do a part of clean up job.

Reply:
I think other than that, the cleaning up doesn't sound necessary except for testing. The whole stats objects should fate-share with the main script.